openapi-cli/src/test/java/org/ballerinalang/openapi/CodeGeneratorTest.java

```diff
@@ -175,11 +200,13 @@ private String getStringFromGivenBalFile(Path expectedServiceFile, String s) thr
         return expectedServiceContent;
     }
 
-    private void deleteGeneratedFiles(String filename) {
-        File serviceFile = new File(resourcePath.resolve(filename).toString());
-        File schemaFile = new File(resourcePath.resolve("schema.bal").toString());
-        serviceFile.delete();
-        schemaFile.delete();
+    private void deleteGeneratedFiles(String filename) throws BallerinaOpenApiException {
+        try {
+            Files.deleteIfExists(resourcePath.resolve(filename));
+            Files.deleteIfExists(resourcePath.resolve("schema.bal"));
+        } catch (IOException  e) {
+            throw new BallerinaOpenApiException("No such file exists.");
```

Comment:
Instead of throwing this, put a comment like `//ignore`. Because we cant handle it. Then theres nothing to mention in the `throws` of the function signature.